frontend/appflowy_flutter/integration_test/desktop/database/database_row_page_test.dart

```diff
@@ -22,7 +23,7 @@ import '../../shared/util.dart';
 
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
-
+  RecentIcons.enable = false;
```

Comment:
Please add a comment to explain why `RecentIcons.enable` is set to false. I suggest disabling it in the `setupAll` function and enabling it in the `tearDownAll` function to avoid side effects.

```suggestion

  setUpAll(() {
    RecentIcons.enable = false;
  });

  tearDownAll(() {
    RecentIcons.enable = true;
  });

```

Reply:
Good suggestion，I'll make the changes based on this suggestion